playbooks/collect_instance_statistics/query_elasticsearch.py

```diff
@@ -0,0 +1,107 @@
+#!/home/ubuntu/instance_statistics_venv/bin/python
```

Comment:
@pcockwell, not a good idea to hardcode the path to the python interpreter.

```suggestion
#!/usr/bin/env python3
```

Reply:
Done in next commit